module-4/metascan/metascan.py

```diff
@@ -6,9 +6,9 @@
 from PyPDF2 import PdfReader
 import argparse
 
-def extract_pdf_links(url):
+def extract_pdf_links_from_website(url):
     response = requests.get(url)
-    response.raise_for_status()
+    response.raise_for_status() #HTTP Error
```

Comment:
This comment does not provide any value so far. To improve it you could say:

`# this call handles HTTP errors that arise when an unsuccessful HTTP response code is returned from the previous call, e.g. 400, 401, 500, etc.`

Since this might be a longer sentence, better place the comment above the code line you want to describe with it